### DISCUSSION TITLE
Replacing FastAPI with Rust: Part 4 - A Solution

### DISCUSSION THREAD

**Discussion by dbanty** [dbanty/dylananthony.com#22] [Posts]
In [this blog post](https://dylananthony.com/posts/fastapi-rust-4-a-solution) I finally select a Rust web framework to use in future projects! Have a suggestion for what I should take on in my next post? Leave it below!

**Comment by mosheduminer** [comment]
Cool! Looking at the code in the experiments repo, it seems quite nice!
